force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +109,16 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        String evenOrOddNum;
+
+        if (a == null){
+            evenOrOddNum = null;
+        } else if (Math.mod(a, 2) == 0){
+            evenOrOddNum = 'Even';
+        } else {
+            evenOrOddNum = 'Odd';
+        }
+        return evenOrOddNum; // Replace null with the variable you used to store the result
```

Comment:
good work. Here is a example of doing with a ternary. I thought I would share since you used one earlier. 

 public static String checkEvenOdd(Integer a) {
        if(a == null ){
            return null;
        }
        return Math.mod(a, 2) == 0 ? 'Even' : 'Odd';
    }